components/mediation-admin/org.wso2.carbon.rest.api/src/main/java/org/wso2/carbon/rest/api/CarbonServerConfig.java

```diff
@@ -112,4 +112,8 @@ public String getHost(API api) throws AxisFault {
             throw new AxisFault("http/https transport listeners are required in axis2.xml");
         }
     }
+
+    @Override public String getHost(String transport) throws AxisFault {
+        return null;
```

Comment:
Shall we add a comment mentioning why we are returning null here?

Reply:
+1 resolved